modules/voipaccountbillinglist.php

```diff
@@ -103,6 +104,8 @@ function sessionHandler($item, $name)
 unset($voipaccountlist['order']);
 unset($voipaccountlist['direction']);
 
+$voipownerlist = array_column($voipaccountlist, "owner", "ownerid");
```

Comment:
Użyj Utils::array_column().

Reply:
Jeśli użyję Utils::array_column() to mam duplikaty i skopiowaną tylko jedną kolumnę voipownerlist.

Jak za pomocą Utils::array_column() uzyskać taki efekt:
```
[4] = klient_z_voipem_id4
[8] = klient_z_voipem_id8

```
?

Reply:
Podaj przykład tablicy wejściowej oraz jaki jest oczekiwany efekt dla tablicy wyjściowej. Wówczas będę mógł zweryfikować działanie funkcji Utils::array_column().

Reply:
Wejście:
`Array ( [0] => Array ( [id] => 1 [login] => 814753777 [passwd] => 8jgod9GvbHyMSr38 [ownerid] => 7 [owner] => ASDASDADS 123 [access] => 1 [borough_name] => [district_name] => [state_name] => [city_name] => [street_name] => [street_type] => [location_name] => [location_city_name] => Lubartów [location_street_name] => ul. Lubelska 36B [location_city] => [location_street] => [location_house] => [location_flat] => [location] => 21-100 Lubartów, ul. Lubelska 36B [phone] => Array ( [0] => 814753777 ) ) [1] => Array ( [id] => 3 [login] => 814753779 [passwd] => mK3qY8sIhWHh8HHo [ownerid] => 3 [owner] => DDDD asdasd [access] => 1 [borough_name] => [district_name] => [state_name] => [city_name] => [street_name] => [street_type] => [location_name] => [location_city_name] => SSS [location_street_name] => [location_city] => [location_street] => [location_house] => [location_flat] => [location] => SSS, SSS [phone] => Array ( [0] => 814753779 ) ) [2] => Array ( [id] => 4 [login] => 814753780 [passwd] => cuqHRH5Jt6h6Cv5W [ownerid] => 4 [owner] => KOWALSKI Jane [access] => 1 [borough_name] => [district_name] => [state_name] => [city_name] => [street_name] => [street_type] => [location_name] => [location_city_name] => Lubartów [location_street_name] => ul. Lubelska 36B [location_city] => [location_street] => [location_house] => [location_flat] => [location] => 21-100 Lubartów, ul. Lubelska 36B [phone] => Array ( [0] => 814753780 ) ) [3] => Array ( [id] => 5 [login] => 814753781 [passwd] => D8zMIMujF0ytA0u4 [ownerid] => 8 [owner] => MALINOWSKI Jan [access] => 1 [borough_name] => [district_name] => [state_name] => [city_name] => [street_name] => [street_type] => [location_name] => [location_city_name] => Lubartów [location_street_name] => [location_city] => [location_street] => [location_house] => [location_flat] => [location] => Lubartów, Lubartów [phone] => Array ( [0] => 814753781 ) ) [4] => Array ( [id] => 2 [login] => 814753778 [passwd] => 8jgod9GvbHyMSr38 [ownerid] => 2 [owner] => TEST TEST [access] => 1 [borough_name] => Lubartów [district_name] => lubartowski [state_name] => lubelskie [city_name] => Lubartów [street_name] => Lubelska [street_type] => ul. [location_name] => [location_city_name] => Lubartów [location_street_name] => ul. Lubelska [location_city] => 12617 [location_street] => 11393 [location_house] => 22 [location_flat] => [location] => 21-100 Lubartów, ul. Lubelska 22 [phone] => Array ( [0] => 814753778 ) ) [5] => Array ( [id] => 6 [login] => 814753783 [passwd] => 0AXLMSjJ5lvNuyUF [ownerid] => 9 [owner] => TRUSKAWSKI Krzysztof [access] => 1 [borough_name] => [district_name] => [state_name] => [city_name] => [street_name] => [street_type] => [location_name] => [location_city_name] => Lubartów [location_street_name] => [location_city] => [location_street] => [location_house] => [location_flat] => [location] => Lubartów, Lubartów [phone] => Array ( [0] => 814753783 ) ) [6] => Array ( [id] => 7 [login] => 814753784 [passwd] => 0AXLMSjJ5lvNuyUF [ownerid] => 9 [owner] => TRUSKAWSKI Krzysztof [access] => 1 [borough_name] => [district_name] => [state_name] => [city_name] => [street_name] => [street_type] => [location_name] => [location_city_name] => Lubartów [location_street_name] => [location_city] => [location_street] => [location_house] => [location_flat] => [location] => Lubartów, Lubartów [phone] => Array ( [0] => 814753784 ) ) )`


Oczekiwane (ownerid => ownername):
`Array ( [7] => ASDASDADS 123 [3] => DDDD asdasd [4] => KOWALSKI Jane [8] => MALINOWSKI Jan [2] => TEST TEST [9] => TRUSKAWSKI Krzysztof )`

Reply:
Proszę: https://github.com/lmsgit/lms/commit/6e3880b94d591b6ab17fe8a4d52a5800ec0e2e9c

Reply:
Dzięki. A tak btw jaki jest cel tego by nie używać tej funkcji bezpośrednio?

Reply:
Minimalizacja odchyleń od zgodności z PHP 5.4.

Reply:
Dzięki za wyjaśnienie.